symbulate/distributions.py

```diff
@@ -341,14 +344,30 @@ class Normal(Distribution):
     """
 
     def __init__(self, mean=0.0, var=1.0, sd=None):
+
+        #Note: cleaner way to implement this
+
         if sd is None:
-            self.scale = np.sqrt(var)
+            if (var > 0):
+                self.var = var
```

Comment:
No need to set self.var. It gets overwritten later when the parent __init__ method is called.